README.md

```diff
@@ -10,4 +10,239 @@ TODO: description
 
 ## API Routes
 
-TODO: document API routes
+### Shared Account
+
+TODO: Jeremiah
+
+### Insurance Card
+
+#### `GET /api/insuranceCards?owner=USERNAME` - Get insurance cards by owner
+
+**Returns**
+
+- An array of insurance cards created by user with username `owner` in alphabetical order with respect to their purpose
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
+
+#### `GET /api/insuranceCards?purpose=PURPOSE` - Get insurance cards by purpose
+
+**Returns**
+
+- An array of insurance cards with purpose `PURPOSE` in alphabetical order with respect to their purpose
+
+**Throws**
+
+- `400` if `PURPOSE` is not given
+
+#### `POST /api/insuranceCards` - Create a new insurance card
+
+**Body**
+
+- `subscriber_name` _{string}_ - The insurance card's subscriber
+- `member_id` _{string}_ - The insurance card's member ID
+- `group_number` _{string}_ - The insurance card's group number
+- `plan_number` _{string}_ - The insurance card's plan number
+- `plan_type` _{string}_ - The insurance card's plan type
+- `purpose` _{string}_ - The insurance card's purpose
+- `notes` _{string}_ - Notes about the insurance card
+
+**Returns**
+
+- A success message
+- A object with the created insurance card
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the subscriber_name or purpose is empty or a stream of empty spaces
+
+#### `PATCH /api/insuranceCards/:insuranceCardId?` - Edit the information of an insurance card
```

Comment:
I have mixed feelings about the `?` on the route path. What do others think?

Personally I think it just invites more bugs in the backend. I'm fine either way as long as we are consistent though. If we do leave it we will need another error case for when the id is undefined (same comment for all similar routes below).

Reply:
A question mark means the value it follows is optional, right? so you bring up a good point - we would need to cover for it being undefined. I have a `?` in two of my routes, but I agree that it would be easier overall to avoid these errors in the first place. I'm fine with deleting all `?` from my routes.